README.md

```diff
@@ -1,2 +1,26 @@
 # slsa-on-github
 Achieving SLSA 2+ on GitHub
+
+
+Example usage
+```
+name: My Workflow
+on: [push, pull_request]
+jobs:
+  build:
+    steps:
+      # Add build steps here
+      # Upload it to use in the next step
+      - name: Upload build
+        uses: 'actions/upload-artifact@v2'
+        with:
+          name: main
+          path: main
+  provenance:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: asraa/slsa-on-github@master
```

Comment:
are we going to keep the various slsa actions in the same repo? If so, maybe create a new directory called `slsa-provenance` and move the Dockerfile/action.yaml in there; and do `uses: asraa/slsa-on-github/slsa-provenance@bla`
Long-term I assume we'll have different repos, like `slsa-provenance-golang`, correct?